spec/lib/spree/core/mail_interceptor_spec.rb

```diff
@@ -5,8 +5,8 @@
 # We'll use the OrderMailer as a quick and easy way to test. IF it works here
 # it works for all email (in theory.)
 describe Spree::OrderMailer do
-  let(:order) { Spree::Order.new(email: "customer@example.com") }
-  let(:message) { Spree::OrderMailer.confirm_email(order) }
```

Comment:
this is not PR specific Matt, this is breaking in master. I am fixing it now in a separate PR :+1: